src/services/arrayUtils.ts

```diff
@@ -0,0 +1,3 @@
+export function arrayDistinct<T>(arr: T[]): T[] {
```

Comment:
Not sure is this is **service** or just a **helper** 🤔

Reply:
I see we have multiple utils inside the service folder, we should probably consider what is the correct location.